pom.xml

```diff
@@ -20,38 +20,77 @@
 
   <groupId>com.datastax.oss</groupId>
   <artifactId>java-driver-shaded-guava</artifactId>
-  <version>25.2-jre-SNAPSHOT</version>
-  <packaging>bundle</packaging>
+  <version>25.1-jre-graal-sub-1-SNAPSHOT</version>
+  <packaging>jar</packaging>
 
   <name>Shaded Guava artifact for use in the DataStax Java driver for Apache Cassandra®</name>
   <description>Shaded Guava artifact for use in the DataStax Java driver for Apache Cassandra®</description>
   <url>https://github.com/datastax/java-driver-shaded-guava</url>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <!-- Note: When upgrading the version, evaluate the following:
-           0. Change the version of the module to match the guava version.
-           1. Update the Export-Package and Import-Package elements in maven-bundle-plugin to match the values in the MANIFEST
-              for the version of guava being upgraded to. -->
       <version>25.1-jre</version>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.code.findbugs</groupId>
+          <artifactId>jsr305</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.checkerframework</groupId>
+          <artifactId>checker-qual</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.google.errorprone</groupId>
+          <artifactId>error_prone_annotations</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.google.j2objc</groupId>
+          <artifactId>j2objc-annotations</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>animal-sniffer-annotations</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.graalvm.nativeimage</groupId>
+      <artifactId>svm</artifactId>
+      <version>20.0.0</version>
+      <scope>provided</scope>
     </dependency>
   </dependencies>
 
   <build>
     <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.6.1</version>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
       <plugin>
```

Comment:
I'm using the same technique used in the shaded driver-core module:

1. Create a first shaded jar; this jar is functional except that it is not an OSGi bundle.
2. Clean `target/classes` and unpack the jar there.
3. Generate an OSGi manifest from the shaded classes in `target/classes`.
4. Re-create a jar containing the shaded classes, the generated OSGi manifest, and the transformed pom.

Reply:
Note: this approach also appears to have resolved the overlapping class issue referenced [here](https://github.com/datastax/java-driver-shaded-guava/pull/4#issuecomment-638027201).  It effectively moves the resolution of duplicate classes into the shading process.